src/components/review-list.jsx

```diff
@@ -20,16 +20,22 @@ export default function ReviewList () {
             setIsLoading(false)
         })
     }, [])
+    
+    let navigate = useNavigate()
+    function viewReview (review_id) {
+        setIsLoading(true);
+        navigate(`/reviews/:${review_id}`)
```

Comment:
You can leave out the colon here! I think it looks cleaner in the url to just be reviews/2 etc.